components/PlantDetails/index.js

```diff
@@ -78,7 +104,25 @@ export default function PlantDetails({ plant, handleToggleModal, isDelete, isEdi
         </StyledPlantNeedsContainer>
         <StyledEditDeleteSection>
           <Button buttonText={<FaTrashAlt />} handleButtonFunction={() => handleToggleModal("Delete")}/>
-      </StyledEditDeleteSection>
+        </StyledEditDeleteSection>
+        <StyledTipH3>Related Care Tips:</StyledTipH3>
+
+          <StyledTagContainer>
+              {relatedTips.map((tip) => plant === undefined ? null : (
```

Comment:
Since you don't have a real else case here you could skip the ternary operator and just write: `plant && ...`